prisma/seed.ts

```diff
@@ -28,19 +28,44 @@ async function main() {
     where: { slug: testFamily.slug },
   });
 
+  // eslint-disable-next-line fp/no-let
+  let family: Family;
   if (!isTestFamilyExists) {
-    await prisma.family.create({ data: testFamily });
+    family = await prisma.family.create({ data: testFamily });
     console.log(
       `test family created, test family name is "${testFamily.name}"`,
     );
+  } else {
+    family = isTestFamilyExists;
   }
 
-  const testProject = {
-    name: faker.lorem.word(5),
-    description: faker.lorem.sentence(3),
-  };
-  await prisma.project.create({ data: testProject });
-  console.log(`The project created, The project name is "${testProject.name}"`);
+  const isTestProjectExists = await prisma.project.findFirst();
+  if (!isTestProjectExists) {
+    const testProject = {
+      name: faker.lorem.word(5),
+      description: faker.lorem.sentence(3),
+    };
+    await prisma.project.create({ data: testProject });
+    console.log(
+      `The project created, The project name is "${testProject.name}"`,
+    );
+  }
+
+  const isTestMemberExits = await prisma.member.findFirst();
```

Comment:
you can use prisma.member.count here also (no need to change it, it's for your information)